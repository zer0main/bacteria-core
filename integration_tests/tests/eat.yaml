board_width: 10
board_height: 10
moves_number: 3
units:
    - coordinates: [0, 0]
      mass: 5
      direction: 0
      team: 0
      instruction: 0
