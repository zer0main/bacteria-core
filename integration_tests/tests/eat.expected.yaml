board_width: 10
board_height: 10
units:
    - coordinates: [0, 0]
      mass: 8
      direction: 0
      team: 0
      instruction: 0
